tests/eas/acceptance.py

```diff
@@ -44,6 +44,10 @@ def local_setup(env):
         env.target.execute(
             "echo ENERGY_AWARE > /sys/kernel/debug/sched_features")
 
+    if SET_IS_BIG_LITTLE:
+        is_big_little_sysctl = "/proc/sys/kernel/sched_is_big_little"
+        if (env.target.file_exists(is_big_little_sysctl)):
```

Comment:
Looks good.
At first I wondered why the sched_features one doesn't use this, looks like that's because `write_value` is intended for single-value sysctls etc, whereas sched_features is a list.